doc/Learn/Tutorials/Http/HowTo-Http.md

```diff
@@ -1,12 +1,101 @@
 
-This how-to is currently under construction. It will be available soon.
+When working with a complex application, centralized registration of your API endpoints is a good practice. This allows you to easily change the endpoint for a given service, and to easily add new services.
 
-## Work in progress 🚧
+## Step-by-steps
 
-### Have questions or feedback?
+### 1. Enable HTTP
 
-* Help us shape the documentation for this topic by providing feedback on the Uno.Extensions [repo](https://github.com/unoplatform/uno.extensions/discussions/categories/general)
\ No newline at end of file
+* Call the `UseHttp()` method to register a HTTP client with the `IHostBuilder` which implements `IHttpClient`:
+
+    ```csharp
+    private IHost Host { get; }
+
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            .Configure(host => {
+                host
+                .UseHttp( /* ... */);
+            });
+    ...
+    ```
+
+### 2. Register Endpoints
+
+* The `UseHttp()` method accepts a delegate that is used to configure the HTTP client. The delegate is passed the `IHostBuilderContext` and the `IServiceCollection`:
+
+    ```csharp
+    private IHost Host { get; }
+
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            .Configure(host => {
+                host
+                .UseHttp((context, services) =>
+                    {
+                        // ...
+                    });
+            });
+    ...
+    ```
+
+* Ultimately, your service will be based on the functionality provided by the web API, but the `HttpClient` associated with it will be injected into the constructor of your service implementation. You will make requests to the registered endpoint inside your service implementation. In this case, the service interface will look something like this:
+    ```csharp
+    public interface IShowService
+    {
+        Task<Show> GetShowAsync(SourceFeed sourceFeed = null);
+    }
+    ```
+
+* The `AddClient` method is used to register a client with the service collection. When you use the `AddClient` method, pass in the `IHostBuilderContext`, the type of the service, the type of the client, and the endpoint to use for the client. 
+
+* The endpoint is defined in the `EndpointOptions` class. The `EndpointOptions` class can be configured to use the platform-native HTTP handler. 
+
+    ```csharp
+    private IHost Host { get; }
+
+    protected override void OnLaunched(LaunchActivatedEventArgs args)
+    {
+        var builder = this.CreateBuilder(args)
+            .Configure(host => {
+                host
+                .UseHttp((context, services) =>
+                    {
+                        services
+                           .AddClient<IShowService, ShowService>(context,
+                               new EndpointOptions
```

Comment:
EndpointOptions can also come from appsettings, which would be the preferred way to configure endpoints